h/zccLogging.h

```diff
@@ -0,0 +1,1105 @@
+
+
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+
+  SPDX-License-Identifier: EPL-2.0
+
+  Copyright Contributors to the Zowe Project.
+*/
+
+#ifndef MVD_H_ZCCLOGGING_H_
+#define MVD_H_ZCCLOGGING_H_
+
+#ifndef ZCC_LOG_ID
```

Comment:
All the message IDs are too specific. We can't afford having hundreds of official message IDs. Each of the will have to be documented with explanation, system programmer response, user response. It will be hard to document what we have here. See https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.3.0/com.ibm.zosmfcore.consoles.messages.help.doc/IZUCN013E.html as example.